checkstyle-suppresions.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+        "-//Puppy Crawl//DTD Suppressions 1.0//EN"
+        "http://www.puppycrawl.com/dtds/suppressions_1_0.dtd">
+
+<suppressions>
+  <suppress checks="Javadoc" files="."/>
+  <suppress checks="Indentation" files="."/>
```

Comment:
we should agree as a team what our indentation is going to be and override the default

Reply:
I like the 4 space tabs, maybe we can set it as that.

Reply:
We can do custom checks but there isn't quite a way to leverage default checks and override some rules. 
We could create our own checkstyle xml to leverage both but that would isolate us from future enhancements.

Reply:
that's not ideal. but ok. 2 spaces it is then 😂